.github/terraform/.gitignore

```diff
@@ -0,0 +1,38 @@
+# Local .terraform directories
```

Comment:
We don't but I thought it would be better to have a .gitignore file only for terraform files so as not to change the original one. I can update the original one though if you want.

Reply:
Putting terraform related files into original one is better option 👍